src/app/views/starter-packs/StarterPacks.js

```diff
@@ -3,140 +3,153 @@ import "./StarterPacks.scss"
-        {/*<ExternalLink text="Twitter Web API Reference" to="https://developer.twitter.com/en/docs/twitter-api"/>*/}
-
-        {/*<h4>Google Cloud API</h4>*/}
-        {/*<p>*/}
-        {/*  API that allows users to access Google services such as storage access*/}
-        {/*  and machine-learning.*/}
-        {/*</p>*/}
-        {/*<ExternalLink text="Google Cloud API Reference" to="https://cloud.google.com/apis"/>*/}
-
+          <div className="col-xs-1 col-md-6 resource-component">
+            <h4>PeterPortal Public API</h4>
+            <p>
+              PeterPortal Public API provides software developers easy-access to UC
+              Irvine's publicly available data such as: courses information, professor
+              information, grade distribution, and schedule of classes.
+            </p>
+            <ExternalLink text="Peter Portal API" to="https://api.peterportal.org/" />
+          </div>
+        </div>
       </section>
-      <hr style={{width: "60vw", borderTop: "3px solid white", marginTop: "90px", marginBottom: "40px" }}/>
-      <section id="starter-pack-block">
+      <hr style={{ width: "60vw", borderTop: "3px solid white", opacity: "1", marginBottom: "4rem" }} />
+      <section id="frontend-resources">
         <div className="resource-title">
-          <motion.img
-              src={BlueStar}
-              animate={{ rotate: 360 }}
-              transition={{
-                  duration: 10,
-                  loop: Infinity,
-                  ease: "linear",
-              }}
-              alt="star"
-          />
-          <h3>Other Resources</h3>
-          <motion.img
-              src={PinkStar}
-              animate={{ rotate: -360 }}
-              transition={{
-                  duration: 10,
-                  loop: Infinity,
-                  ease: "linear",
-              }}
-              alt="star"
-          />
+          <h3>Frontend Resources</h3>
         </div>
-        <h4>Frontend Starter Code</h4>
         <p>
-          Frontend web development is the practice of producing HTML, CSS 
-          and JavaScript for a website or web application that a user can 
-          see and interact with.
+          Frontend web development is the practice of producing an interface for a website
+          or web application that a user can see and interact with.
         </p>
-        <div className="external-link-list">
-          <ExternalLink text="Frontend Example" to="https://github.com/derek-ye/frontend-demo"/>
+        <div className="row">
+          <div className="resource-component">
+          {/* <div className="col-xs-1 col-md-6 resource-border resource-component"> */}
+            <h4>Frontend Starter Code</h4>
+            <p>A basic webpage built using HTML, CSS, and JavaScript.</p>
+            <div className="resource-link">
+              <ExternalLink text="Frontend Example" to="https://github.com/derek-ye/frontend-demo"/>
```

Comment:
Can you replace the to attribute to 
https://github.com/HackAtUCI/zothacks-frontend-startercode

However, Derek's repo is still useful for Bootstrap, so you could also make a separate link for the Bootstrap example if you want to.

Reply:
Good catch! Change made.